### PR TITLE
Fix iw netlink command return error

### PR DESCRIPTION
**Pull request by dungru**
https://github.com/sysprog21/vwifi/issues/12

### CODE REVIEW

File: vwifi.c [hunk 1]
```diff
@@ -378,6 +378,25 @@ static int owl_disconnect(struct wiphy *wiphy,
     return 0;
 }
 
+/* Called with the rtnl lock held. */
+static int owl_get_station(struct wiphy *wiphy,
+                           struct net_device *dev,
+                           const u8 *mac,
+                           struct station_info *sinfo)
+{
+    return 0;
```
Comment (jserv) [inline vwifi.c:387]: Add `FIXME` or `TODO` comment.

Comment (dungru) [reply vwifi.c:387]: Sure

File: vwifi.c [hunk 2]
```diff
@@ -378,6 +378,25 @@ static int owl_disconnect(struct wiphy *wiphy,
     return 0;
 }
 
+/* Called with the rtnl lock held. */
```
Comment (jserv) [inline vwifi.c:381]: Should be
> Called when rtnl lock was acquired.

File: vwifi.c [hunk 3]
```diff
@@ -378,17 +378,28 @@ static int owl_disconnect(struct wiphy *wiphy,
...
  */
-static struct cfg80211_ops owl_cfg_ops = {
-    .scan = owl_scan,
-    .connect = owl_connect,
-    .disconnect = owl_disconnect,
-};
+static struct cfg80211_ops owl_cfg_ops = {.scan = owl_scan,
+                                          .connect = owl_connect,
+                                          .disconnect = owl_disconnect,
+                                          .get_station = owl_get_station};
```
Comment (jserv) [inline vwifi.c:402]: Change from `.get_station = owl_get_station` to `.get_station = owl_get_station,` (with trailing comma) for consistent style.

**Review by jserv — CHANGES_REQUESTED** [review]
Append "Close #12" at the end of git commit message.

**Pull request merged**